macros/edr/data_monitoring/monitors_query/table_monitoring_query.sql

```diff
@@ -225,7 +225,7 @@
     bucket_freshness_ranked as (
         select
             *,
-            rank () over (partition by edr_bucket_end order by freshness desc) as rank
+            row_number () over (partition by edr_bucket_end order by freshness is null, freshness desc) as row_number
```

Comment:
Actually found a couple of bugs in the new freshness test:
1. When ordering only by `freshness desc`, if they are nulls they have the top rank.
2. rank is not good actually because it allows duplicate values to return (if several values share the highest freshness).